Celeste.Mod.mm/Mod/Everest/Everest.Content.cs

```diff
@@ -709,25 +709,33 @@ public static class Content {
             public static void Add(string path, ModAsset metadata)
                 => TryAdd(path, metadata);
 
+            private static readonly string[] METADATA_YAML_NAMES = {
```

Comment:
metadata.yaml and multimetadata.yaml are no longer supported, I just forgot to remove them from the type guesser apparently